WalletWasabi.Fluent/Helpers/LinuxStartupHelper.cs

```diff
@@ -17,14 +17,16 @@ public static async Task AddOrRemoveDesktopFileAsync(bool runOnSystemStartup)
 		{
 			string pathToExec = EnvironmentHelpers.GetExecutablePath();
 
+			string silentArg = pathToExec + " --silent";
```

Comment:
string pathToExecWithArgs. Use string format instead of concat.

Reply:
Can't use string format, because we don't have the actual string at compile time, so i can't index it.